toolbox/fdc3-for-web/demo/src/client/da/dummy-desktop-agent.ts

```diff
@@ -55,18 +55,98 @@ window.addEventListener('load', () => {
     socket.emit(DA_HELLO, desktopAgentUUID);
 
     const directory = new FDC3_2_1_JSONDirectory();
-    //await directory.load('/static/da/appd.json');
-    await directory.load('/static/da/local-conformance-2_0.v2.json');
+    await directory.load('/static/da/appd.json');
+    //await directory.load('/static/da/local-conformance-2_0.v2.json');
     const sc = new DemoServerContext(socket, directory);
 
     const channelDetails: ChannelState[] = [
-      { id: 'one', type: ChannelType.user, context: [], displayMetadata: { name: 'THE RED CHANNEL', color: 'red' } },
-      { id: 'two', type: ChannelType.user, context: [], displayMetadata: { name: 'THE BLUE CHANNEL', color: 'blue' } },
       {
-        id: 'three',
+        id: 'fdc3.channel.1',
         type: ChannelType.user,
         context: [],
-        displayMetadata: { name: 'THE GREEN CHANNEL', color: 'green' },
+        displayMetadata: {
+          name: 'Channel 1',
+          color: 'red',
+          glyph: '1',
+          glyphColor: 'white',
```

Comment:
I'm getting a compile error here, which I can fix by changing to this:

```
export type ChannelState = {
  id: string;
  type: ChannelType;
  context: Context[];
  displayMetadata: DisplayMetadata & { glyphColor: string };
};
```

in BroadcastHandler 

but is that just me?

Reply:
... but actually I don't think you're even using `glyphColor`!

Reply:
Gah, you're right. That was an abortive attempt to fix color contrast on channels - before I went and did it properly with a luminance based contrast calculation. Removed it. Getting no build errors this end.